mlir/utils/jenkins/Jenkinsfile

```diff
@@ -1044,11 +1042,10 @@ pipeline {
                     stage("Verify MIGraphX with MLIR") {
                         steps {
                             dir('MIGraphX/build') {
-                                timeout(time: 60, activity: true, unit: 'MINUTES') {
-                                    withEnv(['MIGRAPHX_ENABLE_MLIR=1']) {
-                                        // Verify MLIR unit tests
-                                        sh 'make -j$(nproc) driver test_gpu_mlir'
-                                        sh 'ctest -R test_gpu_mlir'
+                                timeout(time: 240, activity: true, unit: 'MINUTES') {
+                                    withEnv(['MIGRAPHX_MLIR_USE_SPECIFIC_OPS=convolution,fused,dot,attention', 'MIGRAPHX_ENABLE_MLIR_INPUT_FUSION=1', 'MIGRAPHX_ENABLE_MLIR_REDUCE_FUSION=1', 'MIGRAPHX_MLIR_ENABLE_SPLITK=1']) {
+                                        // Verify all Unit tests pass with MLIR
+                                        sh 'make -j$(nproc) test_verify test_gpu_mlir test_gpu_fuse_mlir'
```

Comment:
changed this to only run verification tests. 

running entire test suite with `make check` would also run onnx backend tests and some other tests inside which requires a lot longer time.